src/ga4gh/vrs/extras/localizer.py

```diff
@@ -45,27 +41,25 @@ def __init__(self):
         self._ana_maps = {k: make_name_ac_map(k) for k in assy_name_to_map_name}
 
 
-    def localize_allele(self, allele):
+    def localize_allele(self, allele, assembly_name = "GRCh38"):
+        """Converts VRS Allele's named features to sequence location"""
         # copy input variant and replace location
-        # N.B. deepcopy leads to recursion errors
+        # N.B. deepcopy leads to recursion_errors
```

Comment:
Okay... I'll bite... why the new underscore? `recursion_errors` is to a symbol.

Reply:
I'm actually not sure how an underscore got added. I'll remove